test/extensions/filters/http/cache/simple_http_cache/simple_http_cache_test.cc

```diff
@@ -107,25 +125,25 @@ class SimpleHttpCacheTest : public testing::Test {
 
 // Simple flow of putting in an item, getting it, deleting it.
 TEST_F(SimpleHttpCacheTest, PutGet) {
-  const std::string RequestPath1("Name");
-  LookupContextPtr name_lookup_context = lookup(RequestPath1);
+  const std::string request_path1("Name");
```

Comment:
Same as below: This string should be something like "/name" so it provides a valid path (including the leading slash).